samples/peripheral/lpuart/boards/nrf21540dk_nrf52840.overlay

```diff
@@ -10,6 +10,7 @@
 		req-pin = <39>;
 		rdy-pin = <40>;
 	};
+	status = "okay";
```

Comment:
Nit: can you move this to the beginning of the node? That's where the status property is usually placed so it will be easier to spot it there at first glance.

Reply:
ok

Reply:
Done